controller/src/main/java/io/pravega/controller/store/stream/PravegaTableScope.java

```diff
@@ -39,7 +39,7 @@
  * Each such scope table is protected against recreation of scope by attaching a unique id to the scope when it is created. 
  */
 public class PravegaTableScope implements Scope {
-    private static final String STREAMS_IN_SCOPE_TABLE_FORMAT = "Table" + SEPARATOR + "streamsInScope" + SEPARATOR + "%s";
+    private static final String STREAMS_IN_SCOPE_TABLE_FORMAT = "Table" + SEPARATOR + "%s" + SEPARATOR + "streamsInScope" + SEPARATOR + "%s";
```

Comment:
One clarification. I can see that you are adding another `%s` to the string format, which makes sense, we need an additional one for the scope according to this PR. But, the description says that the table name should be:

```
_system/Table-<scope>-<stream>-<table-name>
```

which has three input strings rather than two, what am I missing?

Reply:
Ok, I think I get this one now. This class is about scopes, there is no stream. The format for the name of the scope table is:

```
_system/Table-<scope>-<table-name>
```

is this right?

Reply:
Yes